lib/public/views/Logs/Logs.js

```diff
@@ -105,9 +105,13 @@ export default class LogModel extends Observable {
             environmentIds = environmentIds !== undefined ? [environmentIds] : [];
         }
 
+        if (this._creationModel?.parentLogId === parentLogId && !this._creationModel?.submitted) {
```

Comment:
Why this?
What if I simply update `runNumbers` in the URL?

Reply:
then you reload page, in consequence this if statement will no be reached because there will be no instance of logCreation model

Reply:
No, I think here you suppose a bit too much things on the context. If at some point when navigating we come here with a new list of run numbers but the same parent log, this will fail.

Reply:
I used stronger condition, method checks also runNumbers, lhcFillsNumber and environmentes